packages/framework/src/Model/Product/Search/Export/ProductSearchExportStructureFacade.php

```diff
@@ -42,6 +42,45 @@ public function createIndexes(OutputInterface $output)
                 ProductElasticsearchRepository::ELASTICSEARCH_INDEX
             );
 
+            $this->elasticsearchStructureManager->createAliasForCurrentIndex(
+                $domainId,
+                ProductElasticsearchRepository::ELASTICSEARCH_INDEX
+            );
+
+            $output->writeln('Index created');
+        }
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    public function createAliases(OutputInterface $output)
+    {
+        foreach ($this->domain->getAllIds() as $domainId) {
+            $output->writeln(sprintf('Creating alias for id %s', $domainId));
```

Comment:
Foreach body violates DRY principle, because same code is in foreach of method createIndexesWithoutAliases - please create private method to avoid this.